master_changes.md

```diff
@@ -46,6 +46,7 @@ users)
 ## Source
 
 ## Lint
+  * Fix extra-files handling when linting packages from repository with tarred repositories enabled, see #5068 [#5639 @rjbou]
```

Comment:
Actually according to the tests you can remove that section. It was also failing when tarring was disabled
```suggestion
  * Fix extra-files handling when linting packages from repositories, see #5068 [#5639 @rjbou]
```

Reply:
true, it was copy pasting :)